main.go

```diff
@@ -77,104 +77,104 @@ func wrapMain() error {
 	app.Action = run
 	app.Version = fmt.Sprintf("%s-%s", version, rev)
 	app.Flags = []cli.Flag{
-		cli.BoolFlag{
-			Name:   "dry-run",
-			Usage:  "do not apply the Kubernetes manifests to the API server",
-			EnvVar: "PLUGIN_DRY_RUN",
+		&cli.BoolFlag{
+			Name:    "dry-run",
+			Usage:   "do not apply the Kubernetes manifests to the API server",
+			EnvVars: []string{"PLUGIN_DRY_RUN"},
 		},
-		cli.BoolFlag{
-			Name:   "verbose",
-			Usage:  "dump available vars and the generated Kubernetes manifest, keeping secrets hidden",
-			EnvVar: "PLUGIN_VERBOSE",
+		&cli.BoolFlag{
+			Name:    "verbose",
+			Usage:   "dump available vars and the generated Kubernetes manifest, keeping secrets hidden",
+			EnvVars: []string{"PLUGIN_VERBOSE"},
 		},
-		cli.StringFlag{
-			Name:   "token",
-			Usage:  "service account's JSON credentials",
-			EnvVar: "TOKEN",
+		&cli.StringFlag{
+			Name:    "token",
+			Usage:   "service account's JSON credentials (drone 1.x)",
```

Comment:
since this will also still work for <1.x, might be confusing to include the `(drone 1.x)` note in the usage here?

Reply:
Ah yes, good catch, that's an artifact from an earlier attempt to fix this, let me remove it.